lib/cretonne/src/ir/mod.rs

```diff
@@ -10,7 +10,7 @@ pub mod jumptable;
 pub mod dfg;
 pub mod layout;
 pub mod function;
-mod builder;
+pub mod builder;
```

Comment:
Rather than making the `builder` module public, could you just re-export the type you need? (I'm guessing it's `InstBuilderBase`).